ddocs/medic-client/views/reports_by_form/reduce.js

```diff
@@ -0,0 +1,3 @@
+function(keys, values) {
```

Comment:
It's evaluating it correctly, the problem is that we need to disabled the `unused` check otherwise jshint complains about literally every map and reduce function (because you define a function and in the context of that file it's never used).

Reply:
Ahh of course.